source/state/store.ts

```diff
@@ -37,7 +37,18 @@ const store: Store = configureStore({
   devTools: process.env.NODE_ENV !== 'production',
 });
 
-persistStore(store);
+const persistStoreFn = () => {
+  try {
+    if (process.env.NODE_ENV === 'test') {
+      return null;
+    }
+    persistStore(store);
+  } catch (error) {
+    console.log(error);
+  }
+};
+
+persistStoreFn();
```

Comment:
this one we can let the way it was before and check if there is no errors removing this function, we should always test the store as persistent because that is the way we use in production so if you test something without persisting the state, it might have some issues later when we test in production

Reply:
Ok! I changed the code structure, and I removed this function.
All tests pass without errors:
![Screen Shot 2022-01-25 at 07 49 32](https://user-images.githubusercontent.com/72104971/150963184-810b548e-c18b-4d18-aa94-34a82b65ad19.png)
But, in console, one error appears (unrelated to tests):
![Screen Shot 2022-01-25 at 07 51 25](https://user-images.githubusercontent.com/72104971/150963417-5bdd5feb-4585-4b76-951b-87f1b6e7103a.png)
Not because I write an incorrect test, but because when store are imported, there is called, with the persistStore(), that need browser for working. For fix this issue, I created this function for use persistStore only in production context (that has the browser running).
In console, jest gave me this suggestion:
![Screen Shot 2022-01-25 at 07 58 42](https://user-images.githubusercontent.com/72104971/150964518-e8a580e3-3566-493d-87ee-e8b4c87cbcf4.png)
And when I ran this command, I has this return:
![Screen Shot 2022-01-25 at 08 01 47](https://user-images.githubusercontent.com/72104971/150964995-cb820dc6-2739-4922-95e0-8c323c5aebca.png)
So, idk how to fix this issue 😕
@amandagonsalves